packages/@okta/vuepress-site/docs/guides/sign-into-mobile-app-redirect/main/android/testapp.md

```diff
@@ -2,4 +2,4 @@
 1. [Configure an emulator](https://developer.android.com/studio/run/emulator) or [connect a physical device](https://developer.android.com/studio/run/device) in Android Studio
 2. Run the application using **Shift+F10** on Windows/Linux, **Control+R** on macOS, or click **Run** (the "Play" button) in the toolbar.
 
-If you followed along with this guide or cloned the [completed sample application](https://github.com/oktadev/okta-android-quickstart), the app with the sign-in button appears.
+If you followed along with this guide or cloned the [completed sample application](https://github.com/okta-samples/okta-android-kotlin-sample), the app with the sign-in button appears.
```

Comment:
This isn't really a helpful statement. We have to assume that they did follow the guide. Better to state the expected behavior so that a developer knows if something went wrong. I'd also leave out the reference to the sample... the only place it's relevant is the part that refers to samples... all it does here is add confusion. Describe what they see and what the flows are. Something like

> After the app opens, tap the Sign In button. The Okta sign-in browser opens over the app. After the user successfully signs in, the browser closes and the app ??does what, what's different, is there a Sign Out button ??